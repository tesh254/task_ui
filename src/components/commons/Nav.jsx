import React from "react";

class Nav extends React.Component {
  render() {
    return (
      <nav className="navbar" role="navigation" aria-label="main navigation">
        <div className="navbar-brand">
          <a style={{ fontWeight: "bold", fontSize: "18pt"}} className="navbar-item" href="https://bulma.io">
            Tasks
          </a>

          <a
            role="button"
            className="navbar-burger"
            aria-label="menu"
            aria-expanded="false"
          >
            <span aria-hidden="true"></span>
            <span aria-hidden="true"></span>
            <span aria-hidden="true"></span>
          </a>
        </div>
      </nav>
    );
  }
}

export default Nav;