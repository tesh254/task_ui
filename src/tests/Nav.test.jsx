import React from "react"
import enzyme, { shallow } from "enzyme"
import toJSON from "enzyme-to-json"
import adapter from "enzyme-adapter-react-16"
import Nav from "../components/commons/Nav"

enzyme.configure({ adapter: new adapter() })

describe("Test Navbar component", () => {
    it("should contain certain tags to show it rendered navbar properly", () => {
        const wrapper = shallow(<Nav />)

        expect(wrapper.find("nav").hasClass("navbar")).toBe(true)
    })

    it("should render correctly", () => {
        const wrapper = shallow(<Nav />)

        expect(wrapper).toMatchSnapshot()
    })
})