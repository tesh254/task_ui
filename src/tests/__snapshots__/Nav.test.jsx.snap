// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Navbar component should render correctly 1`] = `ShallowWrapper {}`;
